webui/src/lib/api/index.js

```diff
@@ -710,6 +710,18 @@ class Retention {
 }
 
 class Setup {
+    async getState() {
+        const response = await apiRequest('/setup_lakefs', {
+            method: 'GET',
+        });
+        switch (response.status) {
+            case 200:
+                return response.json();
+            default:
+                throw new Error('Unknown');
```

Comment:
This error is somewhat uninformative.  Can we include `response` on it somehow (and maybe get the status message into the error string)?